Tools/TestWebKitAPI/Tests/WebKitCocoa/ServiceWorkerBasic.mm

```diff
@@ -2471,7 +2472,7 @@ static bool isTestServerTrust(SecTrustRef trust)
 
     auto chain = adoptCF(SecTrustCopyCertificateChain(trust));
     auto certificate = checked_cf_cast<SecCertificateRef>(CFArrayGetValueAtIndex(chain.get(), 0));
-    if (![adoptNS((NSString *)SecCertificateCopySubjectSummary(certificate)) isEqualToString:@"Me"])
+    if (![adoptNS(bridge_cast(SecCertificateCopySubjectSummary(certificate))) isEqualToString:@"Me"])
```

Comment:
Seems like we must do an `adoptCF` here, not an `adoptNS`. Then we can do `bridge_cast` on the result of `get()`.